Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/FileItem.cs

```diff
@@ -1,42 +1,41 @@
 ﻿using Renci.SshNet.Sftp;
 
-namespace Frends.SFTP.DownloadFiles.Definitions
-{
-    internal class FileItem
-    {
-        public DateTime Modified { get; set; }
+namespace Frends.SFTP.DownloadFiles.Definitions;
 
-        public string Name { get; set; }
+internal class FileItem
+{
+    public DateTime Modified { get; set; }
 
-        public string NameWithMacrosExtended { get; set; }
+    public string Name { get; set; }
 
-        public string FullPath { get; set; }
+    public string NameWithMacrosExtended { get; set; }
```

Comment:
You are not using this property anywhere, so it can be removed.

Reply:
Removed.